webpack.config.js

```diff
@@ -44,7 +44,7 @@ if (production) {
 module.exports = [
   {
     context: __dirname,
-    devtool: production ? undefined : 'eval-source-map',
+    devtool: production ? undefined : 'eval',
```

Comment:
The [webpack docs](https://webpack.github.io/docs/configuration.html#devtool) are pretty clear that `eval` will be much faster than `eval-source-map`. Maybe we could use something like `cheap-module-eval-source-map`, which is faster than `eval-source-map` (according to the docs), but still provides line-level source maps? We'd be giving up column information, but that's probably ok. We can always add these back in during a debug session.

@andymikulski is there a noticeable slow down using `cheap-module-eval-source-map` compared to `eval`?

Alternatively, do we want to make this configurable? Something like `npm run watch -- --no-sourcemaps`?